final_prep/mod_zero_hero.rb

```diff
@@ -1,41 +1,57 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
-
+hero_name = "dog"
+special_ability = "flying"
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
-
+greeting = "hello i am #{hero_name}!"
+catchphrase = "I can do #{special_ability}"
 # Declare two variables - power AND energy - set to integers
-
+power = 2
+energy = 590
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
-
+full_power = power * 500
+full_energy = energy * 150
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
-
+is_human = false
+identity_concealed = true
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+arch_enemies = ["Mail man", "Vaccum", "Neighbor"]
+sidekicks = ["Chew Toy", "Owner", "Fire Hydrant"]
 # Print the first sidekick to your terminal
-
+p sidekicks.first
 # Print the last arch_enemy to the terminal
-
+p arch_enemies.last
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies << "Veternarian"
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+p arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.delete_at(0)
```

Comment:
Another alternative to the `.delete_at` method is the `shift` method.  It doesn't require any arguments.